caas/containers_test.go

```diff
@@ -29,8 +29,19 @@ ports:
 config:
   attr: foo=bar; fred=blogs
   foo: bar
+files:
+  - name: configuration
+    mount-path: /var/lib/foo
+    files:
+      file1: |-
+        [config]
+        foo: bar
 `[1:]
 
+	expectedFileContent := `
+[config]
+foo: bar`[1:]
```

Comment:
newline for `[1:]?

Reply:
yaml specifies "|-" hence last \n stripped
i changed to yaml to avoid confusion